resources/testdriver.js

```diff
@@ -49,6 +49,54 @@
      * @namespace {test_driver}
      */
     window.test_driver = {
+        /**
+         Represents `WebDriver BiDi <https://w3c.github.io/webdriver-bidi>`_ protocol.
+         */
+        bidi: {
+            /**
+             * `log <https://w3c.github.io/webdriver-bidi/#module-log>`_ module.
+             */
+            log: {
+                /**
+                 * `log.entryAdded <https://w3c.github.io/webdriver-bidi/#event-log-entryAdded>`_ event.
+                 */
+                entry_added: {
+                    /**
+                     * Subscribe to the `log.entryAdded` event. This does not add actual listeners. To listen to the
+                     * event, use `on` method.
+                     * @param {{contexts?: null | (string | Window)[]}} props - Parameters for the subscription.
```

Comment:
I think calling the argument `params` instead of `props` aligns better with the BiDi protocol, for anyone who's familiar with it:

```suggestion
                     * @param {{contexts?: null | (string | Window)[]}} params - Parameters for the subscription.
```

There are a bunch of places that `props` is used in this PR, only commenting here in the beginning.